Logix/Physics/IsPointInConvexHull.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FrooxEngine.LogiX.Physics
+{
+    public class IsPointInConvexHull : LogixOperator<bool>
+    {
+        public override bool Content => throw new NotImplementedException();
```

Comment:
😭 i got my hopes up

Reply:
Ah! This slithered in!

Currently working on this feature, not quite read to go tho

Reply:
> 😭 i got my hopes up

yeah sorry lmao i saw the email showing the node name and then i saw the code and got sad